### ISSUE TITLE
17803 Error while loading information from indexer TVmaze. Error: 'expected string or buffer'

### ISSUE THREAD

**Issue by medariox** [pymedusa/Medusa#1656]
```
2016-11-24 12:36:42 ERROR    SHOWQUEUE-ADD :: [a146b96] 17803 Error while loading information from indexer TVmaze. Error: 'expected string or buffer'
Traceback (most recent call last):
  File "/home/orangepi/.sickrage/**********/show_queue.py", line 348, in run
    s = t[self.indexer_id]
  File "/home/orangepi/.sickrage/**********/indexers/indexer_base.py", line 254, in __getitem__
    self._get_show_data(key, self.config['language'])
  File "/home/orangepi/.sickrage/**********/indexers/tvmaze/tvmaze_api.py", line 402, in _get_show_data
    self._get_episodes(tvmaze_id, specials=False, aired_season=None)
  File "/home/orangepi/.sickrage/**********/indexers/tvmaze/tvmaze_api.py", line 237, in _get_episodes
    results = self.API.episode_list(tvmaze_id, specials)
  File "/home/orangepi/.sickrage/lib/pytvmaze/tvmaze.py", line 277, in episode_list
    return [Episode(self, episode) for episode in q]
  File "/home/orangepi/.sickrage/lib/pytvmaze/tvmaze.py", line 553, in __init__
    self.summary = _remove_tags(data.get('summary'))
  File "/home/orangepi/.sickrage/lib/pytvmaze/tvmaze.py", line 25, in _remove_tags
    return re_tag.sub('', text)
TypeError: expected string or buffer
```

Happened while trying to add [MARS](http://www.tvmaze.com/shows/17803/mars).

**Comment by p0psicles** [comment]
Hmm it's errorring in tvmaze lib. That sucks. Will need to investigate why that's happening.

**Comment by ripper1967** [comment]
I tried adding Mars as well on a windows machine using tvmaze and it had an error and posted to a closed  1489 issue.

**Comment by p0psicles** [comment]
Should be fixed by #1706 
![image](https://cloud.githubusercontent.com/assets/1331394/20726853/82e93c3c-b677-11e6-982c-f8ca3dc97afc.png)

**Comment by p0psicles** [comment]
Fixed and merged.
